ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -166,7 +166,7 @@ def upload_exporter_script(self):
             str: absolute path to exporter script
 
         """
-        script_path = generate_exporter_script()
+        script_path = generate_exporter_script(use_configmap=True)
```

Comment:
could you make condition to use configmap only from 4.19

Reply:
I would make it even from 4.17 if this is the 4.17 feature.

Reply:
In 4.18, we can use both the CSV and the configmap. In 4.19, we can use only the configmap for the external mode script.

Reply:
yeah, I meant actually 4.18 and not 4.17.  See my last comment suggestion. I would do it this way.
Or
```python
if ocs_version <= version.VERSION_4_18:
```